src/org/opendatakit/briefcase/ui/reused/DetailsStatusButton.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2018 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.opendatakit.briefcase.ui.reused;
+
+import static java.awt.Color.DARK_GRAY;
+import static java.awt.Color.LIGHT_GRAY;
+
+import java.awt.Font;
+import java.awt.Insets;
+import javax.swing.JButton;
+
+public class DetailsStatusButton implements ButtonProcessing, Comparable<ButtonProcessing> {
```

Comment:
I believe implementing `Comparable<ButtonProcessing>` is not needed because we will use a reference to a static method.

Reply:
TableRowSorter is referencing the column class of the table column.  And as long as that class implements the Comparable interface, there is no need to pass in a comparator.

Reply:
Huh! I didn't notice that... Thanks for explaining :)